### PR TITLE
[IMP] wording consistency

### PR DESCRIPTION
**Pull request by fhe-odoo**
@dbo-odoo check this.

### CODE REVIEW

File: addons/payment/models/payment_acquirer.py [hunk 1]
```diff
@@ -71,9 +71,9 @@ class PaymentAcquirer(models.Model):
         help="Make this payment acquirer available (Customer invoices, etc.)")
     auto_confirm = fields.Selection([
         ('none', 'No automatic confirmation'),
-        ('authorize', 'Authorize the amount and confirm the SO on acquirer confirmation'),
-        ('confirm_so', 'Confirm the SO on acquirer confirmation'),
-        ('generate_and_pay_invoice', 'On acquirer confirmation, confirm the SO, generate the invoice and pay it')],
+        ('authorize', 'Authorize the amount and confirm the SO on acquirer confirmation (capture at shipping)'),
```
Comment (dbo-odoo) [inline addons/payment/models/payment_acquirer.py]: Capture at shipping is not in standard; I would remove that part

**Comment by dbo-odoo** [comment]
Besides my comment it seems ok.

**Comment by fhe-odoo** [comment]
Feel free to correct and commit.

**Comment by dbo-odoo** [comment]
``` python
    auto_confirm = fields.Selection([
        ('none', 'No automatic confirmation'),
        ('authorize', 'Authorize the amount and confirm the SO on acquirer confirmation (capture manually)'),
        ('confirm_so', 'Authorize & capture the amount and confirm the SO on acquirer confirmation'),
        ('generate_and_pay_invoice', 'Authorize & capture the amount, confirm the SO and auto-validate the invoice on acquirer confirmation')],
        string='Order Confirmation', default='confirm_so', required=True)
```
Ok for you?

**Comment by fhe-odoo** [comment]
Sold! (pun intended)

**Comment by dbo-odoo** [comment]
Merged in 743743db287ccde108858b20349faaed38db00c2

**Pull request closed without merging**